### ISSUE TITLE
What about the opposite scenario?

### ISSUE THREAD

**Issue by MichaelBrooks-Version1** [MicrosoftDocs/visualstudio-docs#5845]
That's great as far as it goes, but you haven't addressed the opposite scenario, whereby I wish to exclude all .config files except packages.config . I have been experimenting and so far all the evidence that I have seen suggests that exclude always overrides include, but that contravenes the age old rule whereby a finer granularity declaration always overrides a less granular declaration, so I'm clearly doing something wrong and hence I'm turning to the documentation for help.

**Comment by WilliamAntonRohm** [comment]
@MichaelBrooks-Version1 -- thank you for your feedback.  Are you following a particular docs.microsoft.com article?  Meanwhile, please consider these resources:
- [Visual Studio on Q&A](https://docs.microsoft.com/en-us/answers/products/vs) 
- [https://social.msdn.microsoft.com/Forums/vstudio](https://social.msdn.microsoft.com/Forums/vstudio) 
- [https://www.stackoverflow.com](https://www.stackoverflow.com) 
- [https://www.reddit.com/r/VisualStudio/](https://www.reddit.com/r/VisualStudio)

**Comment by MichaelBrooks-Version1** [comment]
Yes I was. I was on a specific page (long since closed now) and I clicked link to give feedback on that page. It was the page that explains the include and exclude options to msbuild.community.tasks.zip . Are you saying that your feedback links don’t track the URL of the page from which they are generated? That’s a bit of an oversight on someone’s part! Let’s start by fixing that, then you might get more value from feedback comments.

Regards,
Michael

**Comment by WilliamAntonRohm** [comment]
@MichaelBrooks-Version1 -- thank you for clarifying.  Was the initial page [Walkthrough: Use MSBuild](https://docs.microsoft.com/en-us/visualstudio/msbuild/walkthrough-using-msbuild?view=vs-2019#include-exclude-and-wildcards)?

Any "This page" feedback button should have added article-specific text like the following to the new issue's description field; I don't know why it didn't in your case:

```
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f41d57f6-25bf-bc84-7ba8-4cb4ecedac05
* Version Independent ID: 69c3bd43-6c2b-834b-18d2-0555a79961a7
* Content: [Walkthrough: Using MSBuild - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/msbuild/walkthrough-using-msbuild?view=vs-2019)
* Content Source: [docs/msbuild/walkthrough-using-msbuild.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/msbuild/walkthrough-using-msbuild.md)
* Product: **visual-studio-windows**
* Technology: **msbuild**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**
```

**Comment by MichaelBrooks-Version1** [comment]
Me neither – I remember seeing that textbox and deliberately avoiding it as per instructions.
It wasn’t that page (it might have been this one: https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-exclude-files-from-the-build?view=vs-2019), although I see that the page you have found also suffers the same issue and so could be similarly improved.

Regards,
Michael

**Comment by WilliamAntonRohm** [comment]
@MichaelBrooks-Version1 -- thank you for the update.


@ghogen -- please look into this issue.
